backend/fms_core/viewsets/instrument_type.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework import viewsets
+from rest_framework.decorators import action
+from rest_framework.response import Response
+
+from fms_core.models import InstrumentType
+from fms_core.serializers import InstrumentTypeSerializer
+
+from ._utils import _list_keys
+from ._constants import _instrument_type_filterset_fields
+
+class InstrumentTypeViewSet(viewsets.ModelViewSet):
+    queryset = InstrumentType.objects.all()
+
+    serializer_class = InstrumentTypeSerializer
+
+    ordering_fields = (
+        *_list_keys(_instrument_type_filterset_fields),
+    )
+
+    filterset_fields = {
+        **_instrument_type_filterset_fields,
+    }
+
+    @action(detail=False, methods=["get"])
+    def list_names(self, _request):
```

Comment:
I don't think we need that ... or at least not specifically here ... It could be nice to provide a list_name to most endpoints but it would have to be present on each endpoint that has a name not just once here or there. Could you remove it from this PR and make a new ticket to add this feature to most endpoint. One thing that make a good API is that it is consistent.